setup.cfg

```diff
@@ -44,6 +44,7 @@ install_requires =
     sqlalchemy>=1.2.0
     defusedxml>=0.5.0
     uvloop>=0.14.0; sys.platform!="win32" and python_version>="3.7"
+    prometheus-client>=0.10.0
```

Comment:
We do not intend to depend on `prometheus_client`, thus remove this.

Reply:
Ok.